tests/pcen_test.py

```diff
@@ -0,0 +1,74 @@
+import torch
+from asteroid_filterbanks import Encoder, Decoder, STFTFB
+from asteroid_filterbanks.pcen import PCEN, ExponentialMovingAverage
+from torch.testing import assert_allclose
+import pytest
+
+
+@pytest.mark.parametrize("n_channels", [1, 2, 4])
+@pytest.mark.parametrize("batch_size", [1, 10])
+@pytest.mark.parametrize("n_filters", [128, 512])
+@pytest.mark.parametrize("timesteps", [1, 10])
+def test_ema(n_channels, batch_size, n_filters, timesteps):
+    ema = ExponentialMovingAverage(n_channels=n_channels)
+
+    mag_spec = torch.randn(batch_size, n_channels, n_filters, timesteps)
+    initial_state = mag_spec[:, :, :, 0:1]
+
+    out = ema(mag_spec, initial_state)
+    assert out.shape == (batch_size, n_channels, n_filters, timesteps)
+
+
+@pytest.mark.parametrize("n_channels", [1, 2, 4])
+@pytest.mark.parametrize("batch_size", [1, 10])
+def test_pcen(n_channels, batch_size):
+    audio = torch.randn(batch_size, n_channels, 16000 * 10)
+
+    fb = STFTFB(kernel_size=256, n_filters=256, stride=128)
+    enc = Encoder(fb)
+    tf_rep = enc(audio)
+
+    pcen = PCEN(n_channels=n_channels)
+    energy = pcen(tf_rep)
```

Comment:
This example would be great in the docstring, could you please add it?

Reply:
Good idea done.